tests/conftest.py

```diff
@@ -350,7 +352,7 @@ def collect_techsupport(request, duthost):
     # request.node is an "item" because we use the default
     # "function" scope
     testname = request.node.name
-    if request.node.rep_call.failed:
+    if request.config.getoption("--collect_techsupport") and request.node.rep_call.failed:
```

Comment:
As much as I doubt anyone is using all these dumps at the moment, do you think we should set the default to enable just to keep it consistent with the current behavior?